tfx/components/transform/executor.py

```diff
@@ -276,29 +278,45 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     Args:
       input_dict: Input dict from input key to a list of artifacts, including:
         - input_data: A list of type `standard_artifacts.Examples` which
-          should contain two splits 'train' and 'eval'.
+          should contain two splits 'train' and 'eval' if custom splits
+          is not specified in splits_config.
         - schema: A list of type `standard_artifacts.Schema` which should
           contain a single schema artifact.
       output_dict: Output dict from key to a list of artifacts, including:
         - transform_output: Output of 'tf.Transform', which includes an exported
           Tensorflow graph suitable for both training and serving;
         - transformed_examples: Materialized transformed examples, which
-          includes both 'train' and 'eval' splits.
-      exec_properties: A dict of execution properties, including either one of:
+          includes both 'train' and 'eval' splits if custom splits is not
+          specified in splits_config.
+      exec_properties: A dict of execution properties, including:
         - module_file: The file path to a python module file, from which the
           'preprocessing_fn' function will be loaded.
         - preprocessing_fn: The module path to a python function that
           implements 'preprocessing_fn'.
+        - splits_config: A transform_pb2.SplitsConfig instance, providing splits
+          that should be analyzed and splits that should be transformed. Default
+          behavior is analyze the 'train' split (when analyze_splits is not set)
+          and transform both 'train' and 'eval' splits (when transform_splits is
+          not set).
 
     Returns:
       None
     """
     self._log_startup(input_dict, output_dict, exec_properties)
```

Comment:
sync to head, I submitted a change

Reply:
i already synced to head in my last commit and merged them. is anything missing?